front/app/containers/App/Meta.tsx

```diff
@@ -33,13 +33,21 @@ const Meta = () => {
     const settings = tenant.data.attributes.settings;
     const tenantLocales = settings.core.locales;
 
-    // TODO: Fix this the next time the file is edited.
-    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-    const headerBg = homepageLayout.data.attributes.craftjs_json
-      ? Object.values(homepageLayout.data.attributes.craftjs_json).find(
-          (node) => node.displayName === 'HomepageBanner'
-        )?.props.image?.imageUrl
-      : '';
+    const headerBg = (() => {
+      const layoutData = homepageLayout.data.attributes.craftjs_json;
```

Comment:
Are we really sure that the `craftjs_json` is always available? Maybe good to keep the check that was there before, just in case (the types are wrong)? Because `Object.values(undefined)` will crash the app.